### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by halerankin**
Adds much needed naming convention details.

### CODE REVIEW

File: packages/svg-icons/README.md [hunk 1]
```diff
@@ -10,6 +10,22 @@ npm install @fluentui/svg-icons
...
+* `name` - Name of the icon from [assets](../assets) that is all lowercased and underscore separated.
+* `size` - Size of the icon that is one of 16/20/24/28/48. Note that some icons do not have all sizes available yet. Our designers are working to add missing ones to complete the collection.
+* `style` - Style of the icon that is one of `regular`, `filled` or `selector`. See the section below for details.
+
+### Icon styles
+The library offers icons in two styles, `regular` and `filled`
+
+regular | filled
+--------|-------
+![ic_fluent_mail_regular](../../art/ic_fluent_mail_regular.png)|![ic_fluent_mail_filled](../../art/ic_fluent_mail_filled.png)
```
Comment (spencer-nelson) [inline packages/svg-icons/README.md:25]: The icon example here would be mail_24_regular or whatever size you want to refer to. Basically, it's <name>_<size>_<style> and doesn't have the ic_fluent prefix

File: packages/svg-icons/README.md [hunk 2]
```diff
@@ -10,6 +10,22 @@ npm install @fluentui/svg-icons
 
 ## Usage
 
+The library offers icons in SVG format; the icon names are structured as: 
+> `ic_fluent_[name]_[size]_[style]`
+
+* `name` - Name of the icon from [assets](../assets) that is all lowercased and underscore separated.
+* `size` - Size of the icon that is one of 16/20/24/28/48. Note that some icons do not have all sizes available yet. Our designers are working to add missing ones to complete the collection.
+* `style` - Style of the icon that is one of `regular`, `filled` or `selector`. See the section below for details.
```
Comment (spencer-nelson) [inline packages/svg-icons/README.md:18]: selector isn't an option in SVG...just Android

**Review by spencer-nelson — CHANGES_REQUESTED** [review]

File: packages/svg-icons/README.md (hunk 2 above)
Comment (halerankin) [reply packages/svg-icons/README.md]: Removed prefix and selector

File: packages/svg-icons/README.md (hunk 1 above)
Comment (halerankin) [reply packages/svg-icons/README.md]: Removed prefix from file names. Added size property.

**Review by spencer-nelson — APPROVED** [review]

**Pull request merged**